modules/network/variables.tf

```diff
@@ -77,3 +77,9 @@ variable "cidr_subnet_elasticsearch_2" {
   description = "The CIDR block to use for the elasticsearch 2 subnet"
   default     = "10.0.17.0/24"
 }
+
+variable "gateway_endpoints" {
```

Comment:
IMO we should just make this the default.

Does this mean that if I had an s3 bucket in a different region that I'd still be routing through the NAT gateways to get there? Example: platform deployed in us-west-2 but bucket in us-east-1.

Reply:
I agree to make it the default which I did.  I will remove the option to turn it off.  No real reason not to have it.

It does not work cross region, so it will go through the NAT gateway to get there.  So it is only beneficial if your buckets are in the same region.